pages/services/storage/install/index.md

```diff
@@ -87,14 +90,14 @@ EOF
 ```bash 
 :; dcos package install storage --package-version=<VERSION>
 ```
-Follow the instructions and push the docker image just built to a docker registry that is accessible in the cluster.
+Follow the instructions and push the docker image that is build to a docker registry accessible in the cluster.
```

Comment:
"push the newly built docker image to a docker registry..." read better (and uses correct grammar)